network/src/commonMain/kotlin/com/wire/kalium/network/exceptions/KaliumException.kt

```diff
@@ -4,10 +4,11 @@ import com.wire.kalium.network.api.ErrorResponse
 import com.wire.kalium.network.api.message.SendMessageResponse
 
 sealed class KaliumException(val errorCode: Int) : Exception() {
-    class RedirectError(val errorResponse: ErrorResponse, throwable: Throwable?) : KaliumException(errorCode = errorResponse.code)
-    class InvalidRequestError(val errorResponse: ErrorResponse, throwable: Throwable?) : KaliumException(errorCode = errorResponse.code)
-    class ServerError(val errorResponse: ErrorResponse, throwable: Throwable?) : KaliumException(errorCode = errorResponse.code)
-    class GenericError(val errorResponse: ErrorResponse?, throwable: Throwable?) : KaliumException(errorCode = errorResponse?.code ?: 400)
+    class RedirectError(val errorResponse: ErrorResponse) : KaliumException(errorCode = errorResponse.code)
+    class InvalidRequestError(val errorResponse: ErrorResponse) : KaliumException(errorCode = errorResponse.code)
+    class ServerError(val errorResponse: ErrorResponse) : KaliumException(errorCode = errorResponse.code)
+    class GenericError(val errorResponse: ErrorResponse?) : KaliumException(errorCode = errorResponse?.code ?: 400)
```

Comment:
Shouldn't we have the original exception for Generic errors, with a stacktrace and message, etc.?